src/Motion.js

```diff
@@ -217,6 +260,12 @@ function shouldStopAnimation (currentValues, values, currentVelocities) {
     // istanbul ignore if
     if (!Object.prototype.hasOwnProperty.call(values, key)) continue
 
+    if (isArray(values[key]) || isObject(values[key])) {
+      if (shouldStopAnimation(currentValues[key], values[key], currentVelocities[key])) {
```

Comment:
Glad to know I'm not crazy for thinking this. The effect I got when the conditions were flipped was that the animation stopped (or appeared to based on the values). One of the values was stuck and no amount of stepping would increase it.